prow/cmd/tide/config.md

```diff
@@ -17,15 +17,16 @@ The following configuration fields are available:
 * `merge_method`: A key/value pair of an `org/repo` as the key and merge method to override
    the default method of merge as value. Valid options are `squash`, `rebase`, and `merge`.
    Defaults to `merge`.
+* `merge_commit_template`: A key/value pair of an `org/repo` as the key and a map `title: "Go template"`, `body: "Go template"` as value. A `PullRequest` is provided as input to the template (see `PullRequest` type in [tide.go](/prow/cmd/tide.go)). This field and map keys are optional.
```

Comment:
This is a bit hard to follow - maybe:

```
 * `merge_commit_template`: A mapping from `org/repo` to a set of Go templates to use when creating the title and body of merge commits. Go templates are evaluated with a `PullRequest`  (see `PullRequest` type in [tide.go](/prow/cmd/tide.go)). This field and map keys are optional.
```

Reply:
Haha, thank you! The docs are the second hardest thing after variable names 😅